### DISCUSSION TITLE
Understanding different modules and how to use them.

### DISCUSSION THREAD

**Discussion by kss682** [eclipse-edc/Connector#4320] [Q&A] [answered]
Hello, 

I have been trying understand on a high level what each module in spi, core, ext does. Is there any documentation that can be referred to onboard on the functionalities of these modules ? 

[e2e-test](https://github.com/eclipse-edc/Connector/blob/main/system-tests/e2e-transfer-test/control-plane/build.gradle.kts) Before using the modules in here I would like to know why they are used and the relationship with each other.

Thanks

**Comment by ndr-brt** [comment, marked as answer]
Folder structure: https://github.com/eclipse-edc/Connector?tab=readme-ov-file#directory-structure
Modules documentation: https://eclipse-edc.github.io/Connector/autodoc/

Generally speaking, please refer to the documentation site: https://eclipse-edc.github.io/docs

**Reply by kss682** [reply]
Thanks, I was looking for something like the second link. :+1:
